lib/console.js

```diff
@@ -249,6 +252,77 @@ Console.prototype.groupEnd = function groupEnd() {
     this[kGroupIndent].slice(0, this[kGroupIndent].length - 2);
 };
 
+const untablableTypes = [
+  'symbol', 'undefined', 'boolean', 'number', 'string',
+];
+
+const untablableValues = [undefined, null, NaN, true, false];
+
+const untablable = (value) =>
+  Object.keys(value).length > 20 ||
```

Comment:
This check has to be last. Otherwise it will fail for `null` and `undefined`. Please add a test for this as well.